vault/identity_store_entities.go

```diff
@@ -6,6 +6,8 @@ import (
 	"fmt"
 	"strings"
 
+	"k8s.io/utils/strings/slices"
```

Comment:
Can you use our internal strutil package instead?

Reply:
Good point, not sure how I missed that, fixed